public/sass/_variables.dark.scss

```diff
@@ -376,7 +373,7 @@ $panel-editor-tabs-line-color: #e3e3e3;
 $panel-editor-viz-item-bg-hover: darken($sapphire-base, 46%);
 
 $panel-options-group-border: none;
-$panel-options-group-header-bg: $gray-blue;
+$panel-options-group-header-bg: $dark-3;
```

Comment:
noooo, so nice with that gray blue, it gives the dark theme some slight and much needed saturation

Reply:
I agree with the saturation. But I felt panel-bg and gray-blue were so close. It got a bit weird when using variables.